src/main/java/seedu/duke/flashcard/PersonFlashcard.java

```diff
@@ -0,0 +1,52 @@
+package seedu.duke.flashcard;
+
+import seedu.duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class PersonFlashcard extends Flashcard {
+    private String birthDate;
+    private String deathDate;
+
+    public static PersonFlashcard createPersonFlashcard() {
+        Ui ui = new Ui();
```

Comment:
Do we really need to instantiate a new UI every time we create a new flashcard? Considering having the UI object passed in to the createPersonFlashcard() method.

Reply:
Good idea. I will make this change